forge/src/main/java/org/valkyrienskies/mod/forge/mixin/feature/forge_interact/MixinIForgePlayer.java

```diff
@@ -11,6 +11,7 @@
 import org.spongepowered.asm.mixin.Shadow;
 import org.valkyrienskies.mod.common.VSGameUtilsKt;
 import org.valkyrienskies.mod.common.config.VSGameConfig;
+import org.valkyrienskies.mod.common.util.EntityDragger;
 
 @Mixin(IForgePlayer.class)
 public interface MixinIForgePlayer {
```

Comment:
You probably need to change this in the mixinbooster version of this class as well (when you port to 1.20.1)

Reply:
Noted.